build/automake/build.sh

```diff
@@ -28,12 +28,15 @@
 . ../../lib/functions.sh
 
 PROG=automake
-VER=1.15
+VER=1.15.1
 VERHUMAN=$VER
 PKG=developer/build/automake
 SUMMARY="GNU Automake $VER"
 DESC="GNU Automake - A Makefile generator ($VER)"
 
+# make sure test uses gnu bins
+PATH=/usr/gnu/bin:$PATH
```

Comment:
Can you put this just on the `run_testsuite` line so that it doesn't affect the rest of the build process?